Automattic/jetpack
# Fix/jetpack footer inconsistent behavior

Fixes #31969

This is a fallback from #31627 where some inconsistencies between the MyJetpack footer and other pages appeared.
## Proposed changes:

The changes focus on
- Make **Privacy** and **About** links in the footer non-external for connected scenarios.
- Add consistent behavior in the Footer of My Jetpack to the footers in the rest of the Jetpack pages.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Spin up a test site with this branch
- Make sure your site is not connected to Jetpack
- Visit the _Jetpack_ section
- In the footer, check the following links:
  - Jetpack: `https://jetpack.com/`
  - About: `https://jetpack.com/about/`
  - Privacy: `https://automattic.com/privacy/`
  - Terms: `https://wordpress.com/tos/`
  - An Automattic Airline: `https://automattic.com/about/`
- Visit the _My Jetpack_ section
  - In the footer, check the following links:
  - Jetpack: `https://jetpack.com/`
  - About: `https://jetpack.com/about/`
  - Privacy: `https://automattic.com/privacy/`
  - Terms: `https://wordpress.com/tos/`
  - An Automattic Airline: `https://automattic.com/about/`
- Connect your site to Jetpack
- In the footer, check the following links:
  - Jetpack: `https://jetpack.com/`
  - About: `/wp-admin/admin.php?page=jetpack_about`
  - Privacy: `/wp-admin/admin.php?page=jetpack#/privacy`
  - Terms: `https://wordpress.com/tos/`
  - An Automattic Airline: `/wp-admin/admin.php?page=jetpack_about`
- Visit the _My Jetpack_ section
- In the footer, check the following links:
  - Jetpack: `https://jetpack.com/`
  - About: `/wp-admin/admin.php?page=jetpack_about`
  - Privacy: `/wp-admin/admin.php?page=jetpack#/privacy`
  - Terms: `https://wordpress.com/tos/`
  - An Automattic Airline: `/wp-admin/admin.php?page=jetpack_about`
- Click on one of the Learn More buttons in one of the cards like VaultPress Backup
- In the footer, check the following links:
  - Jetpack: `https://jetpack.com/`
  - About: `/wp-admin/admin.php?page=jetpack_about`
  - Privacy: `/wp-admin/admin.php?page=jetpack#/privacy`
  - Terms: `https://wordpress.com/tos/`
  - An Automattic Airline: `/wp-admin/admin.php?page=jetpack_about`
